src/main/scala/KotlinPlugin.scala

```diff
@@ -1,8 +1,8 @@
 package kotlin
 
-import Keys._
-import sbt._
-import sbt.Keys._
+import Keys.*
+import sbt.{Keys as _, *}
+import sbt.Keys.*
```

Comment:
oO

Reply:
This one is kinda fun... 😅 If we don't ignore the `sbt.Keys` import (as part of `import sbt.*`), we get a warning because:
- in Scala 2.12 (current version because sbt), it is shadowed by `kotlin.Keys` (directly accessible as it's in the same package)
- but if we upgrade to Scala 2.13, the behavior is reversed - `sbt.Keys` shadows `kotlin.Keys`
So the warning recommends making the current behavior explicit (by explicitly ignoring the one which currently gets shadowed) to avoid surprises when upgrading Scala version.

In practice, this is really just to avoid the warning - we directly access the members of both `sbt.Keys` and `kotlin.Keys` thanks to the other imports anyway.

Reply:
Did not catch that `package kotlin` which is not present in the path o///